app/assets/javascripts/shortcuts_navigation.coffee

```diff
@@ -14,6 +14,7 @@ class @ShortcutsNavigation extends Shortcuts
     Mousetrap.bind('g m', -> ShortcutsNavigation.findAndFollowLink('.shortcuts-merge_requests'))
     Mousetrap.bind('g w', -> ShortcutsNavigation.findAndFollowLink('.shortcuts-wiki'))
     Mousetrap.bind('g s', -> ShortcutsNavigation.findAndFollowLink('.shortcuts-snippets'))
+    Mousetrap.bind('t', -> ShortcutsNavigation.findAndFollowLink('.shortcuts-find-file'))
```

Comment:
Shouldn't this only be active to on the tree page? Can you also update https://github.com/gitlabhq/gitlabhq/blob/master/app/views/help/_shortcuts.html.haml and the screenshot in https://github.com/gitlabhq/gitlabhq/blob/master/doc/workflow/shortcuts.md